source/exercises/mailroom-part1.rst

```diff
@@ -35,31 +37,32 @@ Write a small command-line script called ``mailroom.py``. This script should be
 * The script should prompt the user (you) to choose from a menu of 3 actions:
   "Send a Thank You," "Create a Report," or "quit."
 
-Sending a Thank You
+Send a Thank You
 -------------------
+.. I changed heading to match title of function.
 
-* If the user (you) selects 'Send a Thank You', prompt for a Full Name.
+* If the user (you) selects 'Send a Thank You,' prompt for a Full Name.
 
-  * If the user types 'list', show them a list of the donor names and re-prompt.
+  * If the user types 'list,' show them a list of the donor names and re-prompt.
   * If the user types a name not in the list, add that name to the data structure and use it.
   * If the user types a name in the list, use it.
-  * Once a name has been selected, prompt for a donation amount.
-  * Turn the amount into a number; it is OK at this point for the program to crash if someone types a bogus amount.
-  * Once an amount has been given, add that amount to the donation history of
+* Once a name has been selected, prompt for a donation amount.
+
+  * Input the amount as a number; it is OK at this point for the program to crash if someone types a bogus amount.
```

Comment:
Should be "Convert the amount into a number"

Reply:
I'm confused. What do you mean by "convert the amount into a number"? Do you mean convert words to a numerals, as in two hundred dollars to 200.00?

Reply:
it refers to programmatically converting the amount entered into a number. aka cast the value to int.